doc/guides/4 - Refinery Extensions/3 - Testing.textile

```diff
@@ -26,6 +26,12 @@ And run
 $ bundle install
 </ruby>
 
+You will need to install phantomjs from http://phantomjs.org/ or through your OS package manager.
+
+<shell>
+$ sudo apt-get install phantomjs
+</shell>
+
```

Comment:
Maybe, but this is potentially the most useful way?  If you can think of a more succinct form I'm keen.